SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -200,31 +203,73 @@ private Task ProcessPendingErrorsAsync()
 
         private Task SendCrashReportsOrAwaitUserConfirmationAsync()
         {
-            return HandleUserConfirmationAsync();
+            bool alwaysSend = ApplicationSettings.GetValue(PrefKeyAlwaysSend, false);
+
+            return Task.Run(async () =>
```

Comment:
sorry for ignorant q, why it is returning a task when it could be 
`private async Task SendCrashReportsOrAwaitUserConfirmation()`
w/o any return Task.Run() in it?

Reply:
I actually have the same question, can we just remove the task.run?